### PR TITLE
Generate exercise READMEs from templates

### PR DESCRIPTION
**Pull request by kytrinyx**
We are working towards making exercises stand-alone.
That is to say: no more generating READMEs on the fly.

This will give maintainers more control over each individual exercise
README, and it will also make some of the backend logic for delivering
exercises simpler.

This does not mean, however, hand-crafting READMEs from scratch.

I wrote some tooling to make it possible to generate the READMEs from
a customizable template.

The template uses the Go text/template package, and the default template
will generate the same READMEs as we have been generating on the fly.

See the documentation in [regenerating exercise readmes][regenerate-docs] for
details.

The READMEs can be generated at any time using a new 'generate' command in configlet. This command has not yet landed in master or been released, but can be built from source in the generate-readmes branch on [configlet][].

Tracking progress of all tracks in: https://github.com/exercism/meta/issues/15

[configlet]: https://github.com/exercism/configlet
[regenerate-docs]: https://github.com/exercism/docs/blob/master/maintaining-a-track/regenerating-exercise-readmes.md

### CODE REVIEW

**Comment by petertseng** [comment]
I would suggest that [a default template](https://github.com/exercism/configlet/pull/24) be used before taking the "Add exercise README templates" commit, but would take the "Generate the exercise READMEs from the new templates" commit by itself if the former requires more discussion.

**Comment by petertseng** [comment]
EBWODP: Previous link was incorrect. I meant to link https://github.com/exercism/configlet/pull/24#issuecomment-314253509 sorry.

**Comment by kytrinyx** [comment]
100% agreed on the default template.

**Comment by kytrinyx** [comment]
This should be good to go provided that the build passes.

**Comment by petertseng** [comment]
sorry, but going to have to regenerate them all. see https://github.com/exercism/rust/pull/314/commits/e74108f50f7ac731c026ae546c121ac5637623bf for why.

```diff
diff --git a/exercises/acronym/README.md b/exercises/acronym/README.md
index f672f7b..34f230c 100644
--- a/exercises/acronym/README.md
+++ b/exercises/acronym/README.md
@@ -34,7 +34,7 @@ haven't already, it will help you with organizing your files.
 
 The [exercism/xrust](https://github.com/exercism/xrust) repository on GitHub is the home for all of the Rust exercises. If you have feedback about an exercise, or want to help implement new exercises, head over there and create an issue. Members of the [rust track team](https://github.com/orgs/exercism/teams/rust) are happy to help!
 
-If you want to know more about Exercism, take a look at the [contribution guide](https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md).
+If you want to know more about Exercism, take a look at the [contribution guide](https://github.com/exercism/docs/blob/master/contributing-to-language-tracks/README.md).
 
 [help-page]: http://exercism.io/languages/rust
 [crates-and-modules]: http://doc.rust-lang.org/stable/book/crates-and-modules.html
```

File: exercises/acronym/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+to pass again.  The test file is located in the `tests` directory.   You can
+also remove the ignore flag from all the tests to get them to run all at once
+if you wish.
+
+Make sure to read the [Crates and Modules](https://doc.rust-lang.org/stable/book/crates-and-modules.html) chapter if you
+haven't already, it will help you with organizing your files.
+
+## Feedback, Issues, Pull Requests
+
+The [exercism/xrust](https://github.com/exercism/xrust) repository on GitHub is the home for all of the Rust exercises. If you have feedback about an exercise, or want to help implement new exercises, head over there and create an issue. Members of the [rust track team](https://github.com/orgs/exercism/teams/rust) are happy to help!
```
Comment (petertseng) [inline exercises/acronym/README.md]: argh (x)

**Comment by kytrinyx** [comment]
> going to have to regenerate them all

That's a good catch!

**Pull request merged**